src/validator/CreditCardValidator.java

```diff
@@ -0,0 +1,62 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardValidator implements CreditCardChecking {
```

Comment:
If we choose to remove the CreditCardChecking interface (describe in review for CreditCardChecking), implements keyword will be unnecessary.